### PR TITLE
modify version to point to branch release-1.2

### PR DESCRIPTION
**Pull request by xinredhat**

### CODE REVIEW

File: .tekton/rhtap-cli-push.yaml [hunk 1]
```diff
@@ -560,7 +560,7 @@ spec:
       - name: IMAGE
         value: $(tasks.build-container.results.IMAGE_URL)
       - name: ADDITIONAL_TAGS
-        value: ["latest"]
+        value: ["1.2"]
```
Comment (flacatus) [inline .tekton/rhtap-cli-push.yaml]: Add here:

1.2 and 1.2.0

Comment (xinredhat) [reply .tekton/rhtap-cli-push.yaml]: fixed

**Review by Roming22 — APPROVED** [review]
lgtm

**Pull request merged**